### PR TITLE
Environment vars

### PR DESCRIPTION
**Pull request by petersowah**
Added dotenv-rails gem to manage env vars

### CODE REVIEW

**Review by edemkumodzi — COMMENTED** [review]
I think we need to update the readme to explain how to use dotenv to manage environment variables. Current readme uses the default Unix environment variables for the setup.

**Comment by petersowah** [comment]
@edemkumodzi  Readme updated with dotenv usage.

File: README.md [hunk 1]
```diff
@@ -21,14 +21,20 @@ Follow instructions [here](https://git-scm.com/downloads) to install the latest
...
+3. Create a _.env_ file in the project root directory and add the values for `ELECTIONS_DB`, `ELECTIONS_DB_TEST`, `ELECTIONS_DB_USERNAME`, `ELECTIONS_DB_PASSWORD` variables. All environment variables go into this file.
+4. Run `rake db:create` to create your database
+5. Run `rake db:migrate` to run all migrations
+>>>>>>> env-config
 6. Run `rails server` to start your server
 
 ## How to run the test suite
 
-To run the test suite, run `rails test`
+To run the test suite, run `rake test`
```
Comment (edemkumodzi) [inline README.md:31]: We are using rails 5 which allows all rake commands to run through the rails command. The recommended  method should be `rails` ref: http://guides.rubyonrails.org/command_line.html & http://blog.bigbinary.com/2016/01/14/rails-5-supports-rake-commands-using-rails.html

File: .gitignore [hunk 2]
```diff
@@ -19,9 +19,6 @@
 # Ignore Intellij files
 .idea
 
-# Ignore Rails environment vars
-/config/initializers/environment_variables.rb
```
Comment (edemkumodzi) [inline .gitignore:23]: We don't need to create environment_variable.rb here anymore?

**Review by edemkumodzi — CHANGES_REQUESTED** [review]
Looks good, I would just change the `rake test` command back to `rails test`

File: .gitignore (hunk 2 above)
Comment (petersowah) [reply .gitignore:23]: yeah... we don't. We only create a .env file and that's it.

File: README.md (hunk 1 above)
Comment (petersowah) [reply README.md:31]: I didn't know about that one. Cool.

File: .gitignore (hunk 2 above)
Comment (edemkumodzi) [reply .gitignore:23]: okay cool

Comment (petersowah) [reply .gitignore:23]: @edemkumodzi I've reverted the changes changes I made earlier, maintaining only the update to env vars only.

**Review by edemkumodzi — APPROVED** [review]

**Pull request merged**